src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -75,4 +97,71 @@ class LocalBackend extends Backend {
       taskOutput.name -> rawValue.map { possiblyAutoConvertedValue(taskOutput, _) }
     }.toMap
   }
+
+  /**
+   * Creates host execution directory, inputs path, and outputs path.  Stages any input files into the workflow-inputs
+   * directory and localizes their paths relative to the container.
+   */
+  override def initializeWorkflow(workflow: WorkflowDescriptor, actualInputs: Map[FullyQualifiedName, WdlValue]): Map[FullyQualifiedName, WdlValue] = {
+    val hostExecutionDirectory = hostExecutionPath(workflow).toFile
+    hostExecutionDirectory.mkdirs()
+
+    val hostExecutionAbsolutePath = hostExecutionDirectory.getAbsolutePath
+    val hostInputsPath = Paths.get(hostExecutionAbsolutePath, "workflow-inputs")
+    hostInputsPath.toFile.mkdir()
+    val hostOutputsPath = Paths.get(hostExecutionAbsolutePath, "workflow-outputs")
+    hostOutputsPath.toFile.mkdir()
+    stageAndLocalizeWorkflowInputs(workflow, actualInputs)
+  }
+
+  /**
+   * {{{cromwell-executions + three-step + 0f00-ba4 = cromwell-executions/three-step/0f00-ba4}}}
+   */
+  private def hostExecutionPath(workflow: WorkflowDescriptor): Path = Paths.get("cromwell-executions", workflow.name, workflow.id.toString)
+
+  /**
+   * {{{/root + cromwell-executions/three-step/0f00-ba4 + call-ps + stdout.out = /root/cromwell-executions/three-step/0f00-ba4/call-ps/stdout.out}}}
+   */
+  private def generateContainerPath(workflow: WorkflowDescriptor, directory: Path, file: Path): Path = {
+    Paths.get("/root", hostExecutionPath(workflow).toString, directory.toFile.getName, file.toFile.getName)
+  }
+
+  /**
+   * Converts paths of any call output `WdlFile`s from host to container paths.  Other `WdlValue`s are left unchanged.
+   */
+  override def localizeCallOutputs(workflow: WorkflowDescriptor, actualOutputs: Map[FullyQualifiedName, WdlValue]): Map[FullyQualifiedName, WdlValue] = {
+    actualOutputs map { case (fqn, value) =>
+      value match {
+        case WdlFile(hostPath) =>
+          fqn -> WdlFile(generateContainerPath(workflow, directory = hostPath.getParent, file = hostPath))
+        case x => fqn -> x
+      }
+    }
+  }
```

Comment:
I need a little hand-holding here... so this is called when a call completes.  Let's say my call completes with output `foo = my_file.txt`

It'd then see that it was indeed a WdlFile, and then transform that into a `/root/cromwell-executions/.../.../my_file.txt"

So this doesn't actually do any localizing?  I'm confused about that...